app/policies/activity_policy.rb

```diff
@@ -89,6 +89,8 @@ def set_commitment?
   end
 
   def destroy?
+    return false if record.fund?
```

Comment:
It could be. I did think about doing it that way, but we often break down the policy methods quite a bit, e.g. this similar pattern on `set_commitment?` right above:

```ruby
    return false if record.fund?
    return true if beis_user?

    record.commitment.nil? && editable_report? && record.organisation == user.organisation
```

I don't have strong feelings either way!

Reply:
Fair! I think this might be preference